README.md

```diff
@@ -27,10 +45,10 @@ git clone https://github.com/catppuccin/alacritty.git ~/.config/alacritty/catppu
 ```yaml
 import:
   # uncomment the flavour you want below:
-  - ~/.config/alacritty/catppuccin/catppuccin-mocha.yml
-#  - ~/.config/alacritty/catppuccin/catppuccin-macchiato.yml
-#  - ~/.config/alacritty/catppuccin/catppuccin-frappe.yml
-#  - ~/.config/alacritty/catppuccin/catppuccin-latte.yml
+  - ~/.config/alacritty/catppuccin/themes/catppuccin-mocha.yml
+#  - ~/.config/alacritty/catppuccin/themes/atppuccin-macchiato.yml
```

Comment:
`attpuccin` D:

Reply:
atpûccín 🥖🥐
I'll push a fix the next week